data_model/src/merkle.rs

```diff
@@ -93,7 +93,7 @@ impl<T> MerkleTree<T> {
     }
 
     /// Return the `Hash` of the root node.
-    pub const fn root_hash(&self) -> HashOf<Self> {
+    pub fn root_hash(&self) -> HashOf<Self> {
```

Comment:
Not happy about this change, but we will be adding a whole lot more `const` when `const new is stable`.